common/subject_types/babylon_subject_types.txt

```diff
@@ -0,0 +1,51 @@
+﻿subject_type_protectorate = {
+	allow_change_country_flag = no
+	use_overlord_map_color = yes #no
+	use_overlord_ruler = no
+	annex_on_country_formation = no
+	can_start_own_diplomatic_plays = yes
+	breaks_if_subject_not_protected = yes
+	join_overlord_wars = no
+	can_have_subjects = yes
+	overlord_must_be_higher_rank = yes
+	overlord_must_be_same_country_type = no
+	use_for_release_country = yes
+	gives_prestige_to_overlord = yes
+	subservient_to_overlord = yes
+	convoy_contribution = 0.5
+		
+	diplomatic_action = protectorate
+	
+	country_type_change_alternatives = {
+		subject_type_tributary
+	}
+	
+	can_change_subject_type_from = {
+		subject_type_customs_union
+	}
+		
+	valid_overlord_country_types = {
+		recognized
+		colonial
+	}
+	
+	valid_subject_country_types = {
+		recognized
+		colonial
+		unrecognized
+	}	
+
+	valid_overlord_ranks = {
+		great_power
+		major_power
+		minor_power
+	}
+	
+	valid_subject_ranks = {
+		major_power #wasn't possible before
```

Comment:
Why exactly do we do this? This also enables the player to protecterate AI major, so the change is pretty huge.

Reply:
We can change ai behavior to have a -100 acceptance of being a subject just to ensure it won't happen. This is primarily to resolve various issues surrounding GP-subject relationships ranging from the Japan situation last game to Swapping India's subject status to aforementioned issues regarding releasables.